eZ/Publish/Core/Limitation/NewSectionLimitationType.php

```diff
@@ -165,4 +158,45 @@ public function valueSchema()
     {
         throw new \eZ\Publish\API\Repository\Exceptions\NotImplementedException(__METHOD__);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getCriterionByTarget(APILimitationValue $value, APIUserReference $currentUser, ?array $target): CriterionInterface
+    {
+        if (empty($targets)) {
+            throw new InvalidArgumentException('$targets', 'Must contain objects of type: Section');
+        }
+
+        if ($this->doEvaluate($value, $target)) {
+            return new MatchAll();
+        } else {
+            return new MatchNone();
+        }
+    }
+
+    /**
+     * Returns true if given limitation value allows all given sections.
+     *
+     * @param \eZ\Publish\API\Repository\Values\User\Limitation $value
+     * @param array|null $targets
+     *
+     * @return bool
+     *
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException
+     */
+    private function doEvaluate(APILimitationValue $value, array $targets = null): bool
```

Comment:
nitpick: you can skip the null on targets as `evaluate()` also checks if empty and throws if they are